src/theme/inputs.styles.js

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+
+const InputWrapper = styled.div`
+  input,
+  textarea,
+  select {
+    padding: var(--small-space) var(--smallest-space);
```

Comment:
change padding to `padding: var(--small-space);` like you originally had in the TextInput component